app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java

```diff
@@ -105,6 +102,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 break;
             case R.id.logout:
                 logOut();
+                break;
+            case R.id.manage_subscriptions:
+                Intent i = new Intent(this, ManageSubscriptionsActivity.class);
+                startActivity(i);
```

Comment:
@zsofinnemeth you dont need the intent anymore, just instantuate it in the parameters